src/StoreApi/SchemaController.php

```diff
@@ -95,6 +96,16 @@ protected function initialize() {
 			$this->schemas[ BillingAddressSchema::IDENTIFIER ],
 			$this->schemas[ ErrorSchema::IDENTIFIER ]
 		);
+		$this->schemas[ CartExtensionsSchema::IDENTIFIER ]   = new CartExtensionsSchema(
+			$this->extend,
+			$this->schemas[ CartItemSchema::IDENTIFIER ],
+			$this->schemas[ CartCouponSchema::IDENTIFIER ],
+			$this->schemas[ CartFeeSchema::IDENTIFIER ],
+			$this->schemas[ CartShippingRateSchema::IDENTIFIER ],
+			$this->schemas[ ShippingAddressSchema::IDENTIFIER ],
+			$this->schemas[ BillingAddressSchema::IDENTIFIER ],
+			$this->schemas[ ErrorSchema::IDENTIFIER ]
+		);
```

Comment:
Why initiate it with all of those classes? the ones in Cart should be enough. Unless you're explicitly using it in the class, I don't think you should include it.

Reply:
It's because `CartExtensionsSchema` inherits `CartSchema` - I actually reviewed this and found a better way of doing it. Committing now.